roles/openshift_logging/tasks/install_logging.yaml

```diff
@@ -195,6 +195,7 @@
     openshift_logging_elasticsearch_cpu_request: "{{ openshift_logging_es_ops_cpu_request }}"
     openshift_logging_elasticsearch_nodeselector: "{{ openshift_logging_es_ops_nodeselector if outer_item.0.nodeSelector | default(None) is none else outer_item.0.nodeSelector }}"
     openshift_logging_elasticsearch_storage_group: "{{ [openshift_logging_es_ops_storage_group] if outer_item.0.storageGroups | default([]) | length == 0 else outer_item.0.storageGroups }}"
+    openshift_logging_es_cluster_size: "{{ openshift_logging_es_ops_cluster_size | int }}"
```

Comment:
this won't impact the handler, let me think of something else we can do... we likely need to adjust the handler's logic to use the appropriate var depending on which cluster its looking at

Reply:
@jcantrill we likely want to have a different variable for `openshift_logging_es_cluster_size` here https://github.com/openshift/openshift-ansible/blob/2a76e10bf05c85b0567e95391310112a4cecede6/roles/openshift_logging_elasticsearch/tasks/restart_cluster.yml#L15

and then we want to set that depending on which value we have for `_cluster_component` https://github.com/openshift/openshift-ansible/blob/2a76e10bf05c85b0567e95391310112a4cecede6/roles/openshift_logging_elasticsearch/tasks/restart_cluster.yml#L7

it would be either `es` or `es-ops` i believe, we can use that to figure out which variable to denote as the cluster size